tests/test_fortune.py

```diff
@@ -0,0 +1,17 @@
+from sarathi.til import print_fortune
+
+def test_fortune():
+    ''' Tests the Fortune Command '''
+    k,l = print_fortune()
+    if l:
+        print('--------------------Faiure--------------------------')
```

Comment:
You should not be printing anything like this in a test. Instead, rely on `pytest`'s own error reporting.

Also, looking at your function call, you are not returning a tuple. `k, l = print_fortune()` will not work anyway. This is because you either return a string, or you return the error output. This is why it is instead better to create your own exception and deal with that.